elastic/kibana
# [maps] synchronize map views in dashboard and canvas

Fixes https://github.com/elastic/kibana/issues/123731

PR adds "Synchronize map movement" action. When clicked, the action opens "Synchronize map movement" modal, allowing users to configure movement between map panels.

<img width="300" alt="Screen Shot 2022-06-24 at 12 39 16 PM" src="https://user-images.githubusercontent.com/373691/175645520-c26c1132-f5b6-48b8-9fea-b0f4bc1a49b9.png">

<img width="300" alt="Screen Shot 2022-06-24 at 12 39 24 PM" src="https://user-images.githubusercontent.com/373691/175645731-c42b7e95-b40d-4f57-851c-d00b2efdb8b8.png">

To test
* create dashboard with multiple maps
* Pan and zoom in one map. Verify all maps move. Pan and zoom in another map.
* Open action and disable synchronization between all panels or a few panels. Verify movement synchronization is disabled as expected.
